### DISCUSSION TITLE
Gas simulation in "Unified Particle Physics for Real-Time Applications"

### DISCUSSION THREAD

**Discussion by Misju** [InteractiveComputerGraphics/PositionBasedDynamics#191] [Q&A]
Thanks for your excellent work! I notice that you have cited "Unified Particle Physics for Real-Time Applications", but I can`t find an implementation for gas simulation in the way mentioned in the article. Is that true? Looking forward to your reply.

**Comment by janbender** [comment]
I think you mean the smoke simulation in the paper, right? No, that is not implemented.
